CONTRIBUTING.md

```diff
@@ -1,9 +1,40 @@
 # Contributing to Atomic Reactor
 
 If you're reading this, it means that you want to contribute to Atomic Reactor. Great! Here are some tips to make sure your pull request gets accepted.
-Note: there are quite a few tools mentioned through this tutorial. To get them all, including Atomic Reactor dependencies, you can run `pip install -r requirements-devel.txt`.
+Note: there are quite a few tools mentioned through this tutorial. To get them all, including Atomic Reactor dependencies, see the [Environment preparation](#environment-preparation) section.
+
 Also note: while this seems to be an awfully long text, chances are you adhere to most of it just because you're a great developer. And don't worry! Us, core devs, must send code through pull requests, too (well, assuming it isn't an obvious oneliner).
```

Comment:
Ugh... I've never realized how arcane and odd this looks until now. We should probably remove it. (definitely not required for this PR)

Reply:
I removed the whole section (not touching the title).